nixos/modules/services/databases/redis.nix

```diff
@@ -40,7 +40,12 @@ in
       enable = mkOption {
         type = types.bool;
         default = false;
-        description = "Whether to enable the Redis server.";
+        description = ''
+          Whether to enable the Redis server. Note that the NixOS module for
+          Redis disables kernel support for Transparent Huge Pages (THP),
+          because this features causes major performance problems for Redis,
+          e.g. (https://redis.io/topics/latency).
```

Comment:
This should probably also be added to the changelog - maybe as an additional phrase in the paragraph from https://github.com/NixOS/nixpkgs/pull/67845.

Reply:
Yes, that is true. I'll add it ASAP.